INSTALL.md

```diff
@@ -134,4 +134,55 @@ target_link_libraries(foo PRIVATE ${OPENTELEMETRY_CPP_LIBRARIES})
 
 ## Build instructions using Bazel
 
-TBD
+### Prerequisites
+
+- A supported platform (e.g. Windows, macOS or Linux).
+Refer to [Platforms Supported](./README.md#supported-development-platforms)
+for more information.
+- A compatible C++ compiler supporting at least C++11.
+Major compilers are supported.
+Refer to [Supported Compilers](./README.md#supported-c-versions) for more information.
+- [Git](https://git-scm.com/) for fetching opentelemetry-cpp source code from repository.
+To install Git, consult the [Set up Git](https://help.github.com/articles/set-up-git/)
+guide on GitHub.
+- [Bazel](https://www.bazel.build/) for building opentelemetry-cpp API,
+SDK with their unittests. We use 3.7.2 in our build system.
```

Comment:
Can we somehow omit `3.7.2.` here. It's 10 seconds to follow the guidelines, even if we don't mention the version explicitly. Maybe suggesting to check what version is being used:
https://github.com/open-telemetry/opentelemetry-cpp/blob/414a7314f7ab343533042f70eaea5810764fdd5d/.bazelversion#L1

Something like: **Please install the exact version of bazel mentioned in `.bazelversion`** ?


mgolovanov@xps:/mnt/c/work/opentelemetry-cpp.main$ bazel build //...
ERROR: The project you're trying to build requires Bazel 3.7.2 (specified in /mnt/c/work/opentelemetry-cpp.main/.bazelversion), but it wasn't found in /usr/bin.

You can install the required Bazel version via apt:
  sudo apt update && sudo apt install bazel-3.7.2

Reply:
Yeah, I think we should likely encourage folks to install/use Bazelisk, as it can auto-correct versions on the fly and leads to a much smoother developer experience when you move back/forth between different bazel projects.